### ISSUE TITLE
confused about doctest_mark_exe

### ISSUE THREAD

**Issue by firefoxwang** [xflr6/graphviz#172]
【environments】:
|  requirement   | version  |
|  :----: | :----: |
| windows | 10 |
| python | 3.8.3 |
|notebook|6.0.3|
|python-graphviz|0.20|
|Graphviz|3.0.0 |

A nameError  ` NameError :name 'doctest_mark_exe' is not defined ` rasied  when I go through the [Documentation](https://graphviz.readthedocs.io) step by step  . And I  couldn't find any documentation about why I need this [`doctest_mark_exe()`](https://github.com/xflr6/graphviz/blob/4aa4d2441552d72925388e7ce2232c818e91d751/docs/conf.py#L57) 。
When I read the notebook examples provided by the  Documentation, I also did not find the use of  `doctest_mark_exe()`。
But this method is frequently used in the user guide and API  Reference documentation.

Can anyone help? Thanks very match!

**Comment by xflr6** [comment]
`doctest_mark_exe()` is a  custom function that is called in the doctest examples to mark examples that use the `dot` binary so the tests can be skipped or xfailed when run in environments where they are not available (similar to https://docs.python.org/3/library/doctest.html#directives, e.g. https://docs.python.org/3/library/doctest.html#doctest.SKIP):

https://github.com/xflr6/graphviz/blob/4aa4d2441552d72925388e7ce2232c818e91d751/conftest.py#L25-L30

In other words, as documentation readers, we can ignore them. I am not sure if there is a better way to implement this.

**Comment by firefoxwang** [comment]
@xflr6  Now I think I got it. 
Thanks for the super detailed explanation above!

**Comment by xflr6** [comment]
Better documented that the reader should skip these lines in b85a7a8664a2050b5d10f5616d01cf26ac9b5e9a.
